src/app/(admin)/admin/settings/rooms/[id]/page.tsx

```diff
@@ -1,16 +1,51 @@
+"use client";
 import AdminButton from "~/app/_components/admin/Button";
+import BackButton from "../../../tasters/@show/_backButton";
 import React from "react";
 import Link from "next/link";
+import { api } from "~/trpc/react";
+import { useRouter } from "next/navigation";
```

Comment:
This and line 14 can be removed

Reply:
I'm using the "api" one, but I deleted "useRouter" and "LoadingSpinner".